lib/web3-contracts.js

```diff
@@ -61,6 +42,36 @@ export function useTokenDecimals(symbol) {
   return tokenDecimals.get(symbol)
 }
 
+export function useConnectorWeight() {
+  const marketMakerContract = useKnownContract('MARKET_MAKER')
+  const [address] = getKnownContract('COLLATERAL_TOKEN')
+  
+  const [reserveRatio, setReserveRatio] = useState()
+  const [collateralToken, setCollateralToken] = useState()
```

Comment:
I would not store as a state or return the result of `marketMakerContract.getCollateralToken`. It's very misleading because we already have a collateral token (wxDAI in our deployed example), and what is being stored here is a an array with a boolean, virtual balance and supply, and the reserve ratio.